script/run-cypress-tests-circle.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "$(find src -name '*.cypress.spec.js' | wc -l)" -eq 0 ]; then
```

Comment:
should this be a script in `package.json`? I know that the script is being used in the `.circleci` config, but that doesn't always show up in searches, and the npm script would make it easier to run locally.

Reply:
This script will most likely only be used in the `.circleci` config, but I can add it to `package.json` for searching purposes.